.github/workflows/delete-gh-pages-folder.yml

```diff
@@ -5,22 +5,32 @@ on: delete
 jobs:
   delete-folder:
     runs-on: ubuntu-latest
+    if: github.event.ref_type == 'branch'
```

Comment:
According to this, we need the brackets: https://docs.github.com/en/actions/learn-github-actions/expressions#example-expression-in-an-if-conditional

Reply:
Good spot.